Makefile

```diff
@@ -1,9 +1,14 @@
 has_sauce := $(SAUCE_USERNAME)
+no_browser_tests := $(NO_BROWSER_TESTS)
+ifdef no_browser_tests
+test-env := 
+else
 ifndef has_sauce
 test-env := test-local
 else
 test-env := test-sauce
 endif
+endif
```

Comment:
This is a makefile so indentation would break the make.
however, if we were to indent it, it would look like

```
ifdef no_browser_tests
        test-env :=
else
        ifndef has_sauce
                test-env := test-local
        else
                test-env := test-sauce
        endif
endif
```